src/test/build-static.test.ts

```
@@ -0,0 +1,347 @@
+/**
+ * Tests de Build Statique - Phase 2
+ * Teste la génération HTML multilingue et l'optimisation des assets
+ */
+
+import { describe, test, expect, beforeAll, afterAll } from 'vitest';
+import { readFile, access } from 'fs/promises';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)